### PR TITLE
Mese_Land by HexBubble, winniepee0, realyg, Elvis26, DiamondPlane

### PR DESCRIPTION
**Pull request by HexyBubbled**

### CODE REVIEW

**Comment by olliy1or** [comment]
add license

File: mese_map/map.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+team.1.color = red
+team.1 = red
+team.1.pos = (-100,-57,101)
+team.2.pos = (-100,-57,-101)
+team.2.color = blue
+author = HexBubble
```
Comment (Panquesito7) [inline mese_map/map.conf:6]: You should mention all those who made the map.

**Review by Panquesito7 — COMMENTED** [review]
Some parts of the map in the screenshot are unloaded. Please fix this.

File: mese_map/map.conf (hunk 1 above)
Comment (HexyBubbled) [reply mese_map/map.conf:6]: Thanks!!!!!!!!!!!!!!!!!!!!!!!

File: mese_map/map.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+team.1.color = red
+team.1 = red
+team.1.pos = (-100,-57,101)
+team.2.pos = (-100,-57,-101)
+team.2.color = blue
+author = HexBubble, winniepee0, realyg, Elvis26, DiamondPlane
```
Comment (Panquesito7) [inline mese_map/map.conf:6]:
```suggestion
author = HexBubble, winniepee0, realyg, Elvis26 and DiamondPlane 
```

**Comment by Panquesito7** [comment]
> add license

License has been added now. See https://github.com/MT-CTF/maps/pull/181/commits/c6f41c3da878a77a90af24489b0e6dd8ff019893 commit.

**Comment by HexyBubbled** [comment]
house?

**Comment by Axaxax88** [comment]
In my opinion the map is so artificial and similar to Abandoned isles.

Also, I found some mistakes or rare things:
1) Why there's a hole here?
screenshot_20201222_161117

2) Why some blocks are diggable and some are undiggable here?
screenshot_20201222_161155
screenshot_20201222_161206

3) Diggable diamond block? You can't craft diamonds from it, so I'm not sure how useful it is.
screenshot_20201222_161313

**Comment by JostP** [comment]
Issues:
- almost everything is indestructible
- similar to unpopular Abandoned isles map
- almost impossible to capture
- issues from Axaxax88's review

To be honest, I don't think this map is suitable for CTF. But let's ask discord community what they think about it.

**Comment by JostP** [comment]
This map won't be added in current state. Tell me if you want to redo it, but I don't think any changes can make this map be merged.

File: mese_map/map.conf [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+team.2.pos = (-100,-57,-101)
+team.2.color = blue
+author = HexBubble, winniepee0, realyg, Elvis26 and DiamondPlane 
+rotation = z
+h = 140
+team.2 = blue
+name = Mese Map
+initial_stuff = default:pick_steel,default:torch 10,default:cobble 60
+r = 115
+license = CC BY 4.0
```
Comment (Panquesito7) [inline mese_map/map.conf:13]: IMHO, [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) is better.

Comment (JostP) [reply mese_map/map.conf:13]: > IMHO, [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) is better.
It is better, but as long as it is open source license, it is fine.

**Pull request closed without merging**